packages/connect-form/src/components/advanced-options-tabs/tls-ssl-tab/tls-client-certificate.tsx

```diff
@@ -36,7 +36,7 @@ function TLSClientCertificate({
           description={'Learn More'}
           disabled={disabled}
           id="tlsCertificateKeyFile"
-          label="Client Certificate (.pem)"
+          label="Client Certificate Bundle (.pem)"
```

Comment:
What about aligning this name with the param (`tlsCertificateKeyFile`) right away? 
```suggestion
          label="Client Certificate and Key (.pem)"
```

I believe the confusion here would be more that is not clear that a key can be put there, the fact that is a bundle is kinda also in the `.pem` extension already.

Reply:
@addaleax do you have any suggestions here?

Reply:
I think `and Key` sounds okay, yeah. It’s probably not possible to explain that they should be concatenated in the limited space here – can/should we do something like tooltips on hover?

Reply:
DS wasn't super happy when we proposed that, but yeah technically we could